### PR TITLE
Update Dotty standard library version

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**

### CODE REVIEW

**Comment by anatoliykmetyuk** [comment]
@lihaoyi do you think the CI failure is related to the PR? Any ideas of why it might not work on different JVM versions?

File: scalalib/worker/src/ZincWorkerImpl.scala [hunk 1]
```diff
@@ -345,7 +345,7 @@ class ZincWorkerImpl(compilerBridge: Either[
         loader = getCachedClassLoader(compilersSig, combinedCompilerJars),
         libraryJar = libraryJarNameGrep(
           compilerClasspath,
-          if (isDotty(scalaVersion)) "2.13.0" else scalaVersion
+          if (isDotty(scalaVersion)) "2.13.2" else scalaVersion
```
Comment (lefou) [inline scalalib/worker/src/ZincWorkerImpl.scala:348]: Why is this a hardcoded value at all?

**Comment by lefou** [comment]
#849 fixed this issue in a smarter way.

**Comment by lefou** [comment]
Thank you!

**Pull request closed without merging**